install_scripts/templates/common/ip-address.sh

```diff
@@ -178,3 +179,16 @@ isValidIpv4() {
         return 1
     fi
 }
+
+#######################################
+# Returns the ip portion of an address.
+# Globals:
+#   None
+# Arguments:
+#   ADDRESS
+# Returns:
+#   PARSED_IPV4
+#######################################
+parseIpv4FromAddress() {
+    PARSED_IPV4=$(echo "$1" | grep --only-matching '[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*')
```

Comment:
this is good but we do support ipv6 i think or at least we did in the past. maybe we wanna merge and add a ticket for support?

Reply:
OK